Patch to add idle timeout in pool (should be better to add select 1 on request event as in oslo_db/sqlalchemy/engines.py)
---
diff -r -u monasca-api-1.1.2-orig/monasca_api/common/repositories/sqla/sql_repository.py monasca-api-1.1.2/monasca_api/common/repositories/sqla/sql_repository.py
--- monasca-api-1.1.2-orig/monasca_api/common/repositories/sqla/sql_repository.py	2016-03-25 12:39:41.000000000 +0000
+++ monasca-api-1.1.2/monasca_api/common/repositories/sqla/sql_repository.py	2016-07-27 06:54:18.769445971 +0000
@@ -49,8 +49,15 @@
                         del database_conf['url']
                     url = URL(**database_conf)
 
+            engine_args = {
+              "pool_recycle": 10,
+              'convert_unicode': True,
+              'connect_args': {},
+              'logging_name': None
+            }
+
             from sqlalchemy import create_engine
-            self._db_engine = create_engine(url)
+            self._db_engine = create_engine(url, **engine_args)
 
             self.metadata = MetaData()
 
